app/assets/src/components/views/phylo_tree/PhyloTreeChecks.js

```diff
@@ -1,4 +1,4 @@
-const MIN_READS = 100;
+const MIN_READS = 1;
```

Comment:
Just trying to understand the new limit: does it make sense to have it for 1 read?

Reply:
Setting it to 1 is equivalent to not having a limit at all (other than presence/absence). Everybody is confused by the limit because it is not stated explicitly -- looks like a bug -- so we should remove it.

In the future, once the workflow goes through a redesign, we should reinstate it, but as a warning / a grayed-out checkbox explaining what's going on.